piano-hero/mobile/src/main/java/com/novoda/pianohero/GameMvp.java

```diff
@@ -0,0 +1,36 @@
+package com.novoda.pianohero;
+
+interface GameMvp {
+
+    interface Model {
+
+        void startGame(StartCallback callback);
+
+        RoundViewModel onNotesPlayed(CompletionCallback callback, Note... note);
+
+        interface StartCallback {
+            void onGameStarted(RoundViewModel viewModel);
+        }
+
+        interface CompletionCallback {
+            void onGameComplete();
+        }
+    }
+
+    interface View {
+
+        void showRound(RoundViewModel viewModel);
+
+        void showGameComplete();
+    }
+
+    interface Presenter {
+
+        void onCreate();
+
+        void onNotesPlayed(Note... notes);
```

Comment:
> The Presenter itself shouldn't be known to the View, since it's the "master" of the View/Model.

I'm not sure it matters if it knows it or not. Whats the benefit of "hiding" the presenter?